cli/command/container/opts.go

```diff
@@ -198,6 +199,7 @@ func addFlags(flags *pflag.FlagSet) *containerOptions {
 	flags.BoolVar(&copts.privileged, "privileged", false, "Give extended privileges to this container")
 	flags.Var(&copts.securityOpt, "security-opt", "Security Options")
 	flags.StringVar(&copts.usernsMode, "userns", "", "User namespace to use")
+	flags.StringVar(&copts.cgroupnsMode, "cgroupns", "", "Cgroup namespace to use")
```

Comment:
Can you add an API version annotation here? That way the flag is hidden / invalid if the CLI connects to an older daemon that doesn't support it; this now requires API version 1.41 or up (https://github.com/moby/moby/pull/39208)

```suggestion
	flags.StringVar(&copts.cgroupnsMode, "cgroupns", "", "Cgroup namespace to use")
	flags.SetAnnotation("cgroupns", "version", []string{"1.41"})
```

Reply:
This should probably have the same description as you used in the markdown docs;

```suggestion
	flags.StringVar(&copts.cgroupnsMode, "cgroupns", "", "Cgroup namespace to use\n'host':    Run the container in the Docker host's cgroup namespace\n'private': Run the container in its own private cgroup namespace\n'':        Use the default Docker daemon cgroup namespace specified by the `--default-cgroupns-mode` option")
```

Or (using a raw string);

```suggestion
	flags.StringVar(&copts.cgroupnsMode, "cgroupns", "", `Cgroup namespace to use
'host':    Run the container in the Docker host's cgroup namespace
'private': Run the container in its own private cgroup namespace
'':        Use the default Docker daemon cgroup namespace specified by the "--default-cgroupns-mode" option (default)`)
```